Gems/WhiteBox/Code/Source/SubComponentModes/EditorWhiteBoxTransformMode.cpp

```diff
@@ -52,4 +66,32 @@ namespace WhiteBox
         return false;
     }
 
+    void TransformMode::CreateTranslationManipulators()
+    {
+        AZStd::unique_ptr<AzToolsFramework::TranslationManipulators> translationManipulators = AZStd::make_unique<AzToolsFramework::TranslationManipulators>(
+            AzToolsFramework::TranslationManipulators::Dimensions::Three, AZ::Transform::CreateIdentity(), AZ::Vector3::CreateOne());
+        translationManipulators->Register(AzToolsFramework::g_mainManipulatorManagerId);
+
+        m_manipulator = AZStd::move(translationManipulators);
+    }
+
+    void TransformMode::CreateRotationManipulators()
+    {
+        // AZStd::unique_ptr<AzToolsFramework::RotationManipulators> translationManipulators = AZStd::make_unique<AzToolsFramework::RotationManipulators>(
+        //     AzToolsFramework::TranslationManipulators::Dimensions::Three, AZ::Transform::CreateIdentity(), AZ::Vector3::CreateOne());
+        // translationManipulators->Register(AzToolsFramework::g_mainManipulatorManagerId);
+        
+        // m_manipulator = AZStd::move(translationManipulators);
```

Comment:
update: I believe you need to use a `shared_ptr` for manipulators to help deal with lifetime issues (this is explicit in the interface to all manipulators (see `MakeShared`)